### PR TITLE
Improve instructions for user raised issues

### PR DESCRIPTION
**Pull request by pshipton**
Add a reference to jpackcore and articles which help find the system core.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-problem.md [hunk 1]
```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
 Diagnostic files
 ----------------
 
-If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (javacore, Snap, jitdump, core). The smaller files can be attached to this Issue. The core should be compressed and made available via a file sharing service (Box, Google Drive, etc). If there are privacy concerns please direct email the files to an OpenJ9 committer.
+If a crash (gpf, assert, abort, etc) or OutOfMemory condition, please provide the diagnostic files produced (core, javacore, jitdump, Snap). The smaller files can be attached to this Issue.
```
Comment (keithc-ca) [inline .github/ISSUE_TEMPLATE/user-problem.md:21]: This is missing something: perhaps it should begin "If you experience a crash..."?
"GPF" should be capitalized, and "etc" should be "etc.".

File: .github/ISSUE_TEMPLATE/user-problem.md [hunk 2]
```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
...
+
+The system core, which is different from the javacore and a very important diagnostic, should be first processed with `jpackcore` (except on Windows), see https://eclipse.dev/openj9/docs/tool_jextract/#dump-extractor-jpackcore. The system core, which is likely quite large, should be compressed (`jpackcore` does this), and made available via a file sharing service (Box, Google Drive, OneDrive, etc). If there are privacy concerns please directly email or Slack the files or link/password to an OpenJ9 committer.
```
Comment (keithc-ca) [inline .github/ISSUE_TEMPLATE/user-problem.md:23]: "important diagnostic" -> "important source of diagnostic information"
"etc" -> "etc."

File: .github/ISSUE_TEMPLATE/user-problem.md [hunk 3]
```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
...
+
+There are a couple of articles which are helpful for finding the core file.
```
Comment (keithc-ca) [inline .github/ISSUE_TEMPLATE/user-problem.md:25]: Suggest: "These articles are helpful for finding core files".

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/user-problem.md [hunk 4]
```diff
@@ -18,7 +18,13 @@ Put the problem summary here.
...
+If you experience a crash (GPF, assert, abort, etc.) or OutOfMemory condition, please provide the diagnostic files produced (core, javacore, jitdump, Snap). The smaller files can be attached to this Issue.
+
+The system core, which is different from the javacore and an important source of diagnostic information, should be first processed with `jpackcore` (except on Windows), see https://eclipse.dev/openj9/docs/tool_jextract/#dump-extractor-jpackcore. The system core, which is likely quite large, should be compressed (`jpackcore` does this), and made available via a file sharing service (Box, Google Drive, OneDrive, etc.). If there are privacy concerns please directly email or Slack the files or link/password to an OpenJ9 committer.
+
+The following articles are helpful for finding core files:
+https://community.ibm.com/community/user/wasdevops/blogs/kevin-grigorenko1/2022/11/14/lessons-from-the-field-23-linux-core-dumps
+https://community.ibm.com/community/user/wasdevops/blogs/kevin-grigorenko1/2023/09/19/lessons-from-the-field-33-capturing-core-dumps-on
```
Comment (keithc-ca) [inline .github/ISSUE_TEMPLATE/user-problem.md:27]: Suggest some formatting hints are in order here so the links are more clearly listed.
Please also remove trailing whitespace (i.e. line 46).

Comment (pshipton) [reply .github/ISSUE_TEMPLATE/user-problem.md]: Updated.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**